data/api/client-server/report_content.yaml

```diff
@@ -74,5 +77,13 @@ paths:
             type: object
           examples:
             application/json: {}
-      tags:
-        - Reporting content
+        404:
```

Comment:
The MSC didn't actually mention the status code that should be used (and our bad for not calling this out before accepting), but `404` [was agreed upon](https://github.com/matrix-org/matrix-spec-proposals/pull/2249#discussion_r1124909199) by the SCT and the author in the PR comments.

Reply:
fwiw, process-wise I'm comfortable saying that the MSC saying `M_NOT_FOUND` being the error code strongly implies a 404 status code, though we should be careful to avoid this situation in future MSCs.